pkg/datasource/sample-datasource.go

```diff
@@ -50,21 +47,19 @@ type queryModel struct {
 	Format string `json:"format"`
 }
 
-func (td *SampleDatasource) query(ctx context.Context, query backend.DataQuery) (backend.DataResponse, error) {
+func (td *SampleDatasource) query(ctx context.Context, query backend.DataQuery) (response backend.DataResponse) {
```

Comment:
named responses... mumble mumble.
Given this is a tutorial for people who might never have written GO. Let's not use named responses. WDYT?

Reply:
See original statement with this PR.